Assets/Scripts/Hex/HexVertices.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using UnityEngine;
+
+/// <summary>
+/// A list of Hex types supported by this package.
+/// </summary>
+public enum HexTypes
+{
+    FlatTop,
+    PointyTop
+}
+
+/// <summary>
+/// Holds matrices and angle needed to convert Hex to world coordinates and vice
+/// versa.
+/// </summary>
+public readonly struct Orientation

```

Comment:
Should this be a class? It's being passed around a lot, and my understanding is that structs are passed by value.

Reply:
It's fine for now. If it causes problems later, change it then.